sdcm/tester.py

```diff
@@ -2926,7 +2926,7 @@ def alter_test_tables_encryption(self, upgradesstables=True, stress_command=''):
         """
         append_scylla_yaml = yaml.safe_load(self.params.get("append_scylla_yaml") or '') or {}
 
-        if (scylla_encryption_options := self.params.get('scylla_encryption_options')
+        if ((scylla_encryption_options := self.params.get('scylla_encryption_options'))
```

Comment:
Tricky one :)

Reply:
but once I've seen the CQL command that run, it was quite clear what's need to be fixed